storage/sqlStorages/postgres/translator.go

```diff
@@ -0,0 +1,63 @@
+package postgres
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/cube2222/octosql/execution"
+	"github.com/cube2222/octosql/physical"
+	"github.com/pkg/errors"
+)
+
+//A structure that stores the relation $n -> expression
```

Comment:
add a space between // and comment (this actually happens in quite a few places so you can correct them all)

Reply:
Reformat Code in Goland, it does this automatically ;)